content/blog/2020-11-16-BlackInDataWeek/index.Rmarkdown

```diff
@@ -0,0 +1,115 @@
+slug: blackindataweek
+photo:
+  author: Luana Azevedo
+  url: https://unsplash.com/photos/AHBNGvRTm_A
+---
+
+```{r setup, include=FALSE}
+library(tidyverse)
+library(glue)
+library(gt)
+```
+
+Today marks the first day of a week-long celebration called [`#BlackinDataWeek`](https://blkindata.github.io/). It is a week to amplify Black voices in `#informatics`, `#datascience`, & `#coding` around the world. The goals are to:
+
+1.  Provide community and a support system for Black people in data across the Black diaspora.
+
+2.  Encourage growth in our professional and academic learning paths.
+
+3.  Elevate the voices and experiences of Black people working in data fields.
+
+Read on to find out more about what events are planned, and how you can engage with [`#BlackinDataWeek`](https://blkindata.github.io/).
+
+## Events
+
+```{r echo=FALSE}
+BlackInDataWeek <- tibble::tibble(
+  date = seq(as.Date("2020-11-16"), as.Date("2020-11-21"), 1),
+  hashtag = c("#BlackInDataRollCall",
+              "#BlackInDataJourney",
+              "#BlackInDataSkills",
+              "#BlackInDataViz",
+              "#BlackInDataJustice", 
+              "#BlackInDataMentorship"),
+  purpose = c(
+    "Giving Black people in data a space to introduce themselves and their work. Introducing and valuing intersecting parts of their identities. We welcome contributions from a wide spectrum of Data Fields including but not limited to Informatics, Technology, Data Science, Coding, Social Science and Data Analytics.",
+    "Further fostering community for Black people in data, by encouraging them to share their varied journeys in data.",
+    "Discussing the skills Black people in data have learned, communal sharing of resources and advice for skills development.",
+    "Creating space for Black people in data to share their work in the form of favourite data visualisation images.",
+    "Hosting forums for learning and discussion of bias in the data field (and possible paths to address biases in data).",
+    "Join us for career development and mentorship events!"
+  ),
+  link = c(
+    "https://blkindata.github.io/project/blackindatarollcall/",
+    "https://blkindata.github.io/project/blackindatajourney/",
+    "https://blkindata.github.io/project/blackindataskills/",
+    "https://blkindata.github.io/project/blackindataviz/",
+    "https://blkindata.github.io/project/blackindatajustice/",
+    "https://blkindata.github.io/project/blackindatacommunity/"
+  )
+) 
+
+BlackInDataWeek %>% 
+  mutate(hashtag = glue::glue('[{hashtag}]({link})')) %>%
+  select(-link) %>% 
+  gt::gt() %>% 
+  fmt_markdown(columns = vars(hashtag))
+```
+
+## How to engage in `#BlackInDataWeek`
+
+### Day 1: `#BlackInDataRollCall`
+
+-   If you identify as Black, TODAY join in the roll call by introducing yourself and sharing what you do in data. Include the hashtag [`#BlackInDataRollCall`](https://twitter.com/hashtag/BlackInDataRollCall) and tag [`@BlkInData`](https://twitter.com/BlkInData).
```

Comment:
"TODAY join in the roll call" <- "join in the roll call today" (I'm not a fan of ALL CAPS)